src/main/java/com/demo/Model/Price.java

```diff
@@ -6,5 +6,5 @@
 @Data
 @AllArgsConstructor
 public class Price {
-    Long totalPrice, totalGst, cGst,sGst;
```

Comment:
Can we use all these as seperate fields, please?

Reply:
Updated as per request. Also change the type to Float.